modules/core/shared/src/main/scala/gem/enum/GnirsCrossDispersed.scala

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package enum
+
+import cats.syntax.eq._
+import cats.instances.string._
+import gem.util.Enumerated
+
+/**
+ * Enumerated type for GNRIS Cross Dispersed.
+ * @group Enumerations (Generated)
+ */
+sealed abstract class GnirsCrossDispersed(
+  val tag: String,
+  val shortName: String,
+  val longName: String,
+  val pixelScale: BigDecimal
+) extends Product with Serializable
+
+object GnirsCrossDispersed {
+
+  /** @group Constructors */ case object Sxd extends GnirsCrossDispersed("Sxd", "SXD", "SXD", 0.15)
+  /** @group Constructors */ case object Lxd extends GnirsCrossDispersed("Lxd", "LXD", "LXD", 0.05)
+
```

Comment:
I realize that I may be commenting on old (and fully functional) code, however, it is surprising to specify the pixel scale here as a parameter of the cross-disperser.  The pixel scale is uniquely determined by the camera.  In addition, the SXD prism may be used with either camera, and therefore have either pixel scale.